apps/client/components/About/AboutPage/index.tsx

```diff
@@ -27,17 +24,42 @@ const AboutPage = () => {
   });
 
   return (
-    <div>
-      <ul>
+    <section
+      css={css`
+        ${padding({ x: 27 })}
+      `}
+    >
+      <motion.ul
+        variants={staggerOne}
+        {...framerMocker}
+        css={css`
+          ${gutter({ direction: "vertical", space: 12 })}
```

Comment:
```suggestion
          ${gutter("vertical", 12)}
```
이렇게 축소하여 사용할 수 있습니다! 참고해주세용~

Reply:
어호호호호호!! 감사합니다 
인터페이스를 제대로 이해못하고있었군요

Reply:
[fix(client): gutter 문법 축소](https://github.com/toppingskorea/toppings-client/pull/92/commits/a59edfe378b40865afa6e8c6cc853197a777760f)